lib/test_helpers.ml

```diff
@@ -0,0 +1,27 @@
+open Parsetree
+open Typedtree
+open Types
+
+(* Helper library to parse .mli content and return Types.signature *)
+let compile_interface (content : string) : Types.signature =
+  (* Parses and tokenizes the contents of the .mli file*)
+  let parse_interface content =
+    let lexbuf = Lexing.from_string content in
+    try Parse.interface lexbuf
+    with exn ->
+      failwith ("Error while parsing .mli content: " ^ Printexc.to_string exn)
```

Comment:
Since you're simply rethrowing here, you can remove the `try ... with` and let the initial parse error be reported.